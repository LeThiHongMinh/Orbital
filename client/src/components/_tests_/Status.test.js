import * as React from 'react'
import StatusComponent from '../Status'
import renderer from 'react-test-renderer'

it('renders correctly', () => {
  const tree = renderer.create(<StatusComponent />)
  expect(tree).toMatchSnapshot()
})