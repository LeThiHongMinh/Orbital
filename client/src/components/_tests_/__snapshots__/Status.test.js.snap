// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="status-component fixed left-0 bottom-0 p-4 bg-white shadow-lg rounded-lg w-full max-w-md"
>
  <h3
    className="text-lg font-semibold mb-60"
  >
    Your Progress
  </h3>
  <div
    className="flex justify-between"
  >
    <div>
      <p
        className="text-sm font-medium text-gray-600"
      >
        Lessons Completed
      </p>
      <p
        className="text-xl font-semibold text-gray-900"
      />
    </div>
    <div>
      <p
        className="text-sm font-medium text-gray-600"
      >
        Quizzes Completed
      </p>
      <p
        className="text-xl font-semibold text-gray-900"
      />
    </div>
  </div>
  <div
    className="flex justify-between mt-60"
  >
    <div>
      <p
        className="text-sm font-medium text-gray-600"
      >
        Prototypes Completed
      </p>
      <p
        className="text-xl font-semibold text-gray-900"
      />
    </div>
    <div>
      <p
        className="text-sm font-medium text-gray-600"
      >
        Hours Learned
      </p>
      <p
        className="text-xl font-semibold text-gray-900"
      />
    </div>
  </div>
</div>
`;
